packages/react-components/token-analyzer-preview/library/src/astAnalyzer.ts

```diff
@@ -72,15 +72,50 @@ function processStyleProperty(prop: PropertyAssignment, isResetStyles?: Boolean)
           processNode(childProp.getInitializer(), [...path, childName]);
         }
       });
-    } else if(Node.isCallExpression(node)){
+    } else if (Node.isCallExpression(node) && node.getExpression().getText() === 'createCustomFocusIndicatorStyle') {
+      const focus = `&[data-fui-focus-visible]`;
+      const focusWithin = `&[data-fui-focus-within]:focus-within`;
+      let nestedModifier = focus;
+
+      const passedTokens = node.getArguments()[0];
+      const passedOptions = node.getArguments()[1];
+
+      if (passedOptions && Node.isObjectLiteralExpression(passedOptions)) {
+        passedOptions.getProperties().forEach(property => {
```

Comment:
nit/optimization: since we should only have one option named `selector` we could probably optimize this with `some()` instead of `forEach` and return true when we have a `nestedModifier` set (if we set one). This way we don't continue to iterate after we have found what we need (shouldn't be a huge deal in this specific case.